README.md

```diff
@@ -1,16 +1,115 @@
-# Animal Rescue ♥️😺 ♥️🐶 ♥️🐰 ♥️🐦 ♥️🐹
+# Animal Rescue
 ![Test All](https://github.com/spring-cloud-services-samples/animal-rescue/workflows/Test%20All/badge.svg?branch=master)
 
-Sample app for Tanzu Spring Cloud Gateway tile. 
-Features we demonstrate with this sample app:
+Sample app for VMware's Spring Cloud Gateway commercial products. Features we demonstrate with this sample app:
+
 - Routing traffic to configured internal routes with container-to-container network
 - Gateway routes configured through service bindings
 - Simplified route configuration
 - SSO login and token relay on behalf of the routed services
 - Required scopes on routes (tag: `require-sso-scopes`)
 - Circuit breaker filter
 
-## Deploy to CF
+## Deploy to Kubernetes
+
+The Kubernetes deployment requires you to install [kustomize](https://kustomize.io/). You will also need to install [Spring Cloud Gateway for Kubernetes](https://network.pivotal.io/products/spring-cloud-gateway-for-kubernetes) successfully onto your target Kubernetes cluster.
+
+### Configure Single Sign-On (SSO)
+
+For Animal Rescue sample Single Sign-On (SSO) to work, you will need to create two text files that will be used to create Kubernetes secrets:
+
+* ./frontend/secrets/sso-credentials.txt
+* ./gateway/sso-secret-for-gateway/secrets/test-sso-credentials.txt
+
+Before you start, and for validation, please locate the JWKS endpoint info `jwks_uri` from your SSO identity provider. The endpoint typically exists at:
```

Comment:
We now refer to JWK URI as `jwk-set-uri` in the secrets and codebase. This readme might be outdated since it refers to it as `jwks-uri` all over

Reply:
I'll change all refs to jwk-set-uri.